docs/src/hamiltonians.md

```diff
@@ -1,4 +1,4 @@
-# Developer Documentation
+offdiagonals# Developer Documentation
```

Comment:
```suggestion
# Developer Documentation
```
offdiagonals snuck in here. I'm not sure if we need the `# Development Documentation` header either. It's visible in the sidebar, and this is the only chapter with the header.